topi/python/topi/nn/winograd_util.py

```diff
@@ -131,6 +132,8 @@ def _interpolation_points(degree):
 
     return np.array(in_pts[degree-1], dtype=np.float64)
 
+
+@memoize("topi.nn.winograd_matrices", save_at_exit=False)
```

Comment:
```
-@memoize("topi.nn.winograd_matrices", save_at_exit=False)
+@memoize("topi.nn.winograd_util.winograd_matrices", save_at_exit=False)
```
Shouldn't be like that ?

Reply:
Oh, seems to be just a name, it works either (and it's shorter a bit). Apologize for holding this up.